tests/test_openclhpp.cpp

```diff
@@ -3407,6 +3407,49 @@ void testLinkProgramWithVectorProgramInput(void)
 #endif
 }
 
+void testGetPlatformVersion_1_1(void)
+{
+    cl_platform_id platform = make_platform_id(0);
+
+    clGetPlatformInfo_StubWithCallback(clGetPlatformInfo_version_1_1);
+
+    cl_uint version = cl::detail::getPlatformVersion(platform);
+    TEST_ASSERT_EQUAL_UINT32(0b10000000000000001, version);
```

Comment:
Nitpick: in the `opencl.hpp` code we check for hex constants instead, so it'd be clearer to use them here vs. binary constants.  Same goes for the other new tests, also.

```suggestion
    TEST_ASSERT_EQUAL_UINT32(0x10001, version);
```

Reply:
Changed to hexadecimal.